src/platform-includes/session-replay/configure-sample-rate/javascript.mdx

```diff
@@ -0,0 +1,19 @@
+```javascript {tabTitle: ESM}
+// If you're using one of our integration packages, like `@sentry/angular`,
+// substitute its name for `@sentry/browser` here
+import * as Sentry from "@sentry/browser";
+
+import { Replay } from "@sentry/replay";
+
+Sentry.init({
+  dsn: "___PUBLIC_DSN___",
+
+  // This enables automatic instrumentation (highly recommended), but is not
+  // necessary for purely manual usage
+  integrations: [new Replay()],
```

Comment:
We'll also need to add a CDN tab here, as the setup isn't 100% identical (the namespace for the integration is `Sentry.Integrations.Replay`). As already said below, I can add this in a follow-up PR later if you want.